daemon/main.c

```diff
@@ -578,6 +579,8 @@ static void options(int *argc, char ***argv) {
 		{ "homer",	0,  0, G_OPTION_ARG_STRING,	&homerp,	"Address of Homer server for RTCP stats","IP46|HOSTNAME:PORT"},
 		{ "homer-protocol",0,0,G_OPTION_ARG_STRING,	&homerproto,	"Transport protocol for Homer (default udp)",	"udp|tcp"	},
 		{ "homer-id",	0,  0, G_OPTION_ARG_INT,	&rtpe_config.homer_id,	"'Capture ID' to use within the HEP protocol", "INT"	},
+		{ "homer-enable-rtcp-stats", 0, 0, G_OPTION_ARG_INT,	&rtpe_config.homer_rtcp,	"Enable/disable RTCP stats to Homer (enabled by default if homer server enabled, else disabled)", "0|1"	},
+		{ "homer-enable-ng", 0, 0, G_OPTION_ARG_INT,	&rtpe_config.homer_ng,	"Enable/disable NG to Homer", "0|1"	},
```

Comment:
I'd say those should be boolean values, but I understand the problem with the RTCP switch. You should be able to work around it by using a `gboolean` and still setting it to -1 (or some other magic value), or perhaps by using a `G_OPTION_ARG_CALLBACK`. Alternatively a negated option meaning would also work.

Reply:
Hi, if OK with you I would go for a negated option homer-disable-rtcp-stats and the normal homer-enable-ng, both gboolean.

Reply:
> Hi, if OK with you I would go for a negated option homer-disable-rtcp-stats and the normal homer-enable-ng, both gboolean.

Sounds good